### PR TITLE
Add benchmark plugin

### PR DESCRIPTION
**Pull request by tamirdavid**
I have read the Developer Terms and I hereby accept the terms

### CODE REVIEW

File: plugins/benchmark.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+pluginName: benchmark
+version: v0.1.2
+repository: https://github.com/tamirdavid/benchmark
+maintainers:
+  - tamirdavid
+tag: v0.1.2
```
Comment (eyalbe4) [inline plugins/benchmark.yml:6]: It looks like ythe plugin's repository doesn't have a tag named `v0.1.2`. You'll need to create this tag for the registry to be able to build your pugin.


image

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
We recomment using the letaet releases of the [jfrog-cli-core](https://github.com/jfrog/jfrog-cli-core/releases) and [jfrog-client-go](https://github.com/jfrog/jfrog-client-go/releases) depedencies in your plugin's go.mod file,

File: plugins/benchmark.yml (hunk 1 above)
Comment (tamirdavid) [reply plugins/benchmark.yml:6]: @eyalbe4 added

**Comment by eyalbe4** [comment]
Confrats @tamirdavid!
Your new HFrog CLI Plugin is released and can be installed by running -
```
jf plugin install benchmark
```

**Pull request merged**